src/backend/utils/aborthandler/aborthandler.ts

```diff
@@ -29,7 +29,7 @@ function callAbortController(id: string) {
 }
 
 function callAllAbortControllers() {
-  for (const key in abortControllers.keys()) {
```

Comment:
This was causing native games launched through HP to not be closed when HP was closed